protobuf/avs.proto

```diff
@@ -658,10 +658,22 @@ message DeleteSecretReq {
   string org_id = 3;
 }
 
+message GetSignatureFormatReq {
+  // The wallet address to include in the signature format
+  string wallet = 1;
+}
+
+message GetSignatureFormatResp {
+  // The formatted signature message with server-side values filled in
+  string format = 1;
```

Comment:
The returned variable should be called message, since values are filled into the format. The `format` is misleading.